css/card.css

```diff
@@ -104,13 +110,13 @@ tcg-card .card-subtitle {
   animation: left 1s ease 0s 1;
   animation-fill-mode: forwards;
 }
-#gacha-controls tcg-card.clicked:nth-child(3n + 1) {
+#gacha-controls.pile-display tcg-card.clicked:nth-child(3n + 1) {
```

Comment:
Without this increased specificity, the subtitle text animates on rare+ cards in grid view flip